src/main/java/com/github/batkinson/plugins/PackMojo.java

```diff
@@ -146,6 +151,15 @@ public void execute() throws MojoExecutionException {
             }
         }
 
+        if (passFiles != null) {
+            int pos = 0;
+            for (String passFile : passFiles) {
+                getLog().debug("bypassing " + passFile);
+                packerProps.put(Packer.PASS_FILE_PFX + pos, passFile);
+                pos++;
+            }
+        }
+

```

Comment:
This all looks good. If the pull request included just this, it would be easy to approve.